logger/logger.go

```diff
@@ -64,6 +64,10 @@ type logger struct {
 	Logger kitlog.Logger
 }
 
+func (l *logger) Decorate(keyvals ...interface{}) {
+	l.Logger = kitlog.NewContext(l.Logger).With(keyvals...)
```

Comment:
@kopiczko `Decorate` is basically just another name for `With`. Or do you mean something else?